doc/developing.md

```diff
@@ -20,20 +20,11 @@
 
 Official (yet incomplete) reference: https://llvm.org/docs/BitCodeFormat.html
 
-C++ implementation:
- + Parser:
-   * [Release 4.0](https://github.com/llvm-mirror/llvm/blob/release_40/lib/Bitcode/Reader/BitcodeReader.cpp)
-   * [Release 5.0](https://github.com/llvm-mirror/llvm/blob/release_50/lib/Bitcode/Reader/BitcodeReader.cpp)
-   * [Release 6.0](https://github.com/llvm-mirror/llvm/blob/release_60/lib/Bitcode/Reader/BitcodeReader.cpp)
+Relevant parts of the upstream C++ implementation:
+ + [Parser (13.x)](https://github.com/llvm/llvm-project/blob/release/13.x/llvm/lib/Bitcode/Reader/BitcodeReader.cpp)
```

Comment:
Any reason not to link to the latest LLVM release (14.x) here?

Reply:
Nope, good point!